tsconfig.json

```diff
@@ -51,7 +51,6 @@
     "cypress",
     "cypress.config.ts",
     "**/*.cy.ts",
-    "**/*.cy.tsx",
-    "__tests__"
```

Comment:
This line was causing linting to fail for me. If it works for you, I can add it back